pkg/configurator/methods.go

```diff
@@ -225,3 +225,18 @@ func (c *client) GetFeatureFlags() configv1alpha2.FeatureFlags {
 func (c *client) GetOSMLogLevel() string {
 	return c.getMeshConfig().Spec.Observability.OSMLogLevel
 }
+
+// GetTLSMinimumProtocolVersion returns the minimum supported TLS protocol version
+func (c *client) GetTLSMinimumProtocolVersion() string {
+	return c.getMeshConfig().Spec.Sidecar.TLSMinimumProtocolVersion
+}
+
+// GetTLSMaximumProtocolVersion returns the maximum supported TLS protocol version
+func (c *client) GetTLSMaximumProtocolVersion() string {
+	return c.getMeshConfig().Spec.Sidecar.TLSMaximumProtocolVersion
+}
+
+// GetCipherSuites returns the list of supported TLS cipher suites
+func (c *client) GetCipherSuites() []string {
+	return c.getMeshConfig().Spec.Sidecar.CipherSuites
+}
```

Comment:
Going forward, please do not add methods for retrieving individual fields, directly use `GetMeshConfig().Spec.Sidecar.xyz`